R/across.R

```diff
@@ -455,7 +459,7 @@ across_setup <- function(cols,
   )
 
   if (!inline) {
-    fns <- map(fns, as_function)
+    fns <- map(fns, as_function, call = quote(across()))
```

Comment:
```suggestion
    fns <- map(fns, as_function, call = call(across_if_fn))
```

I think you want this? i.e. if `if_any/all()` go through here